metadata.json

```diff
@@ -46,11 +46,8 @@
     {
       "operatingsystem": "Fedora",
       "operatingsystemrelease": [
-        "22",
-        "23",
-        "24",
-        "25",
-        "31"
+        "32",
+        "33"
```

Comment:
I'm worried https://github.com/voxpupuli/puppet-borg/pull/78 won't support this due to beaker-hostgenerator's lack of F33 support.

Reply:
true, but I did a quick local test and the module seems to support it. So I guess it's fine to add 32/33 here and exclude 33 in #78 ?